Lesson16/lesson16/ezgi/script.js

```diff
@@ -0,0 +1,221 @@
+// 1. Write a function that takes in a number and checks if it's even or odd using if else statement.
+
+function isEvenOrOdd(number) {
+	const reminder = number % 2;
+	// === - strict equality
+	if (reminder === 0) {
+		return 'Even';
+	} else {
+		return 'Odd';
+	}
+}
+
+console.log('Exercise 1', isEvenOrOdd(9), isEvenOrOdd(2));
+// 2. Write a function that takes age as input. If the age is less than 18, print "Sorry, you are not old enough to vote". If the age is between 18 and 65 (inclusive), print "You are eligible to vote". If the age is greater than 65, print "You are a senior citizen".
+function canVote(age) {
+	if (age < 18) {
+		return 'Sorry, you are not old enough to vote';
+	} else if (18 <= age && age <= 65) {
+		return 'You are eligible to vote';
+	} else {
+		return 'You are a senior citizen';
+	}
+}
+
+console.log('Exercise 2', canVote(10), canVote(30), canVote(75));
+
+// 3. Write a function that asks the user to enter two numbers and checks if the first number is greater than, less than, or equal to the second number using if else statement.
+function equalityOfTwoNumbers (number1,number2){
+if(number1>number2){
+    return  'first number is greater than second number';
+}
+else if  (number1===number2){
+    return 'Numbers are equal';
+}
+else{
+    return  'first number is less than second number';
+}
+}
+console.log('Exercise 3' , equalityOfTwoNumbers(-15,-30), ',', equalityOfTwoNumbers(80,1000), ',',equalityOfTwoNumbers(4,4));
+// 4. Write a function that takes in a string and checks if the first letter is uppercase using if else statement.
+function isFirstLetterUppercase(stringToCheck){
+const firstLetter = stringToCheck[0];
+  if (firstLetter === firstLetter.toUpperCase()) {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log('Exercise 4' , isFirstLetterUppercase("Televiision"));
+// 5. Write a function that takes in a number and checks if it's positive, negative, or zero using if else statement.
+function checkTheNumber (number3){
```

Comment:
No need to change, just a comment for the future:
since the number3 is a parameter of the `checkTheNumber` function, it is a local variable. This means that you only have to change inside `checkTheNumber` function that there's no other variable with the name 'number3'.
In other words, you can use word 'number' as function parameter if you like as there are no other variables with name 'number' defined in this function.

Reply:
Got it! :)